internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+MaxRunDuration = '10m' # Default
+# In order to keep database size manageable, Chainlink nodes will run a reaper that deletes completed job runs older than a certain threshold age. `ReaperInterval` controls how often the job pipeline reaper will run.
+#
+# Set to `0` to disable the periodic reaper.
+ReaperInterval = '1h' # Default
+# Determines the age limit for job runs. Completed job runs older than this will be automatically purged from the database.
+ReaperThreshold = '24h' # Default
+# Some jobs write their results asynchronously for performance reasons such as OCR. `JOB_PIPELINE_RESULT_WRITE_QUEUE_DEPTH` controls how many writes will be buffered before subsequent writes are dropped.
+#
+# Do not change this setting unless you know what you are doing.
+ResultWriteQueueDepth = 100 # Default
+
+[FluxMonitor]
+DefaultTransactionQueueDepth = 1 # Default
+# Enable transaction simulation for Flux Monitor.
+SimulateTransactions = false # Default
+
+[OCR2]
+ContractConfirmations = 3 # Default
+BlockchainTimeout = '20s' # Default
+ContractPollInterval = '1m' # Default
+ContractSubscribeInterval = '2m' # Default
+ContractTransmitterTransmitTimeout = '10s' # Default
+DatabaseTimeout = '10s' # Default
+# TODO
+KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000' # Example
+# TODO
+MonitoringEndpoint = 'test-mon-end' # Example
+
+# This section applies only if you are running off-chain reporting jobs.
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h'
+ContractSubscribeInterval = '1m'
+DefaultTransactionQueueDepth = 12
+# Default key bundle ID to use for OCR jobs. If you have an OCR job that does not explicitly specify a key bundle ID, it will fall back to this value.
+KeyBundleID = 'acdd42797a8b921b2910497badc5000600000000000000000000000000000000' # Example
+# Optional URL of OCR monitoring endpoint.
+MonitoringEndpoint = 'test-monitor' # Example
+# Enable transaction simulation for OCR.
+SimulateTransactions = false # Default
+# The default sending address to use for OCR. If you have an OCR job that does not explicitly specify a transmitter address, it will fall back to this value.
+TransmitterAddress = '0xa0788FC17B1dEe36f057c42B6F373A34B014687e' # Example
+TraceLogging = false # Default
+
+# P2P supports multiple networking stack versions. You may configure `[P2P.V1]`, `[P2P.V2]`, or both to run simultaneously.
+# If both are configured, then for each link with another peer, V2 networking will be preferred. If V2 does not work, the link will
+# automatically fall back to V1. If V2 starts working again later, it will automatically be preferred again. This is useful
+# for migrating networks without downtime. Note that the two networking stacks _must not_ be configured to bind to the same IP/port.
+#
+# All nodes in the OCR network should share the same networking stack.
+[P2P]
+IncomingMessageBufferSize = 10 # Default
+OutgoingMessageBufferSize = 10 # Default
```

Comment:
Again, see OCR docs: https://github.com/smartcontractkit/libocr/blob/master/networking/ocr_endpoint_v1.go#L38

Reply:
Probably all the OCR/OCR2/P2P internal config variables are ** advanced/dev only **